zipline-kotlin-plugin/build.gradle.kts

```diff
@@ -5,20 +5,28 @@ import com.vanniktech.maven.publish.MavenPublishBaseExtension
 
 plugins {
   kotlin("jvm")
+  kotlin("kapt")
+  id("com.github.gmazzo.buildconfig")
   id("org.jetbrains.dokka")
   id("com.vanniktech.maven.publish.base")
+  id("com.github.johnrengelman.shadow")
 }
 
 dependencies {
-  compileOnly(projects.ziplineKotlinPluginHosted)
+  compileOnly(kotlin("compiler"))
```

Comment:
When I tried it, I depended on compiler-embedded and dropped the shadowJar

Reply:
(this also works!)

Reply:
Oh fun. Let's do that. I'll PR.